plugins/pencilblue/templates/admin/login.html

```diff
@@ -38,7 +38,7 @@
                                 </div>
                                 <input type="button" class="btn btn-lg btn-primary btn-block" value="^loc_LOGIN^" onclick="login()">
                                 <div class="forgot_password">
-                                    <a href="javascript:forgotPassword()">^loc_FORGOT_PASSWORD^</a>
+                                    <a href="javascript:forgotPassword()">I forgot my password</a>
```

Comment:
This was Brian's change https://github.com/cbdr/pencilblue/commit/7a57e17c33ad0658dc25e874362dfade55b8cdc8